src/App.js

```diff
@@ -23,7 +44,19 @@ export default function App() {
     setActivePage('cropdetails')
   }
 
-  function handleClickBack() {
+  function handleClickList() {
     setActivePage('croplist')
   }
+
+  function handleClickFavorites() {
+    setActivePage('favorite')
+  }
```

Comment:
```suggestion
  function handleClickPage(page) {
    setActivePage(page)
  }
```

you could create only one dynamical function